CONTRIBUTING.md

```diff
@@ -12,7 +12,7 @@ That said, some basic guidelines, which you are free to ignore :)
 
 Want to lurk about and see what others are doing with Logstash?
 
-* The irc channel (#logstash on irc.freenode.org) is a good place for this
+* The irc channel (#logstash on irc.freenode.org) is a good place for this. IS THIS STILL ACTIVE? 
```

Comment:
we can keep it but maybe lower on the list, as no one from the team hangs around there as far as I know, so it's mostly community driven

Reply:
Based on my testing, `irc.freenode.org` appears to be defunct. `http://irc.freenode.net/` was suggested as an alternate, but also appears to be defunct.